src/scss/style.scss

```diff
@@ -0,0 +1,92 @@
+.module {
+    color: #625F5F;
+    border-radius: 2px;
+    padding: 15px;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #C8C4C3;
+    min-height: 100vh;
+
+    .wrapper {
```

Comment:
Adding this selector as children will generate extra long selector. Is there any value to add extra "body" in front of every selector from here to the end?